src/OfflineSupport/synctoast.tsx

```
@@ -0,0 +1,36 @@
+      {/* Message */}
+      <p className="text-sm font-medium">{message}</p>
+
+      {/* Button */}
+      {showButton && (
+        <button
+          onClick={() => {
+            navigate(`/facility/${facilityId}/settings/sync-status`);
+            dismiss(toastId);
+          }}
+          className="ml-3 text-sm cursor-pointer text-green-500 font-medium underline underline-offset-4 hover:text-green-800 hover:underline"
+        >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)